api/chaos/experimentation/v1/experiment.proto

```diff
@@ -23,10 +23,11 @@ message Experiment {
     STATUS_STOPPED = 5;
   }
 
-  uint64 id = 1;
-  // The experiment configuration specific to the type of experiment
+  // An unique identifier of an experiment run.
+  string run_id = 1;
+  // The experiment configuration specific to the type of experiment.
   google.protobuf.Any config = 2;
-  // The time when the experiment should start. If unspecified, defaults to 'now'
+  // The time when the experiment should start. If unspecified, defaults to 'now'.
```

Comment:
When would this not be set? Don't we automatically populate it?

Reply:
do server experiments not use the CreateExperiment flow?

Reply:
They use the flow. The proto is a part of the public API and a caller may not provide `startTime` field. The doc explains what server does if `startTime` is not provided by a caller.

Reply:
Discussed offline. This was the right point - removed a piece about not the value of `startTime` not being provided.